docs/install-on-aws.md

```diff
@@ -0,0 +1,233 @@
+
+For a gentler introduction to Swarm, try the [Evaluate Swarm in a sandbox](https://docs.docker.com/swarm/install-w-machine/) page.
+
+<This example doesn't use Amazon's EC2 Container Service (ECS)>
+
+## Prerequisites
+
+- An Amazon Web Services (AWS) account
+- Familiarity with AWS features and tools, such as:
+  - EC2 Dashboard
+  - VPC Dashboard
+  - VPC Security groups
+  - Connecting to an EC2 instance using SSH
+
+## Establish basic network security
+
+You create basic network security by restricting the types of inbound traffic that can reach the hosts on your network. To accomplish this, you create a security group and add rules to it. Each rule specifies the type, protocol, port range, and source of the traffic that is allowed to reach your hosts. This security group excludes all other inbound traffic. It also has a rule that allows all outbound traffic.
+
+This one-size-fits-all security group is suitable for an example like this one. For a production environment, you would probably create multiple security groups with more restrictive rules for traffic to each type of host. To establish network security for your production environment, consult a network security expert.
+
+Important: You do not need to create a VPC. New EC2 instances use a default VPC. ***When you create the following security group, associate it with the default VPC.***
+
+From your AWS home console, click **VPC - Isolated Cloud Resources**. Then, in the VPC Dashboard that opens, navigate to **Your VPCs**.
+
+Check the **VPC CIDR** of the default VPC (e.g., "172.30.0.0/16"). If the CIDR does not contain the "172.30.0.0" dotted quad, update the following rules with the actual dotted quad. Leave the "/24" portion of the CIDR unchanged.
+
+Navigate to **Security Groups**. Create a security group named "Docker Swarm Example" with the following inbound rules. The **Allows** column explains what each rule allows and is just for your reference.
+
+| Type            | Protocol  | Port Range | Source        | Allows            |
+| --------------  | -----     | -----      | -----         | -----             |
+| SSH             | TCP       | 22         | 0.0.0.0/0     | SSH connection    |
+| HTTP            | TCP       | 80         | 0.0.0.0/0     | Container images  |
+| Custom TCP Rule | TCP       | 2375       | 172.30.0.0/24 | Non-TLS traffic   |
+| Custom TCP Rule | TCP       | 8500       | 172.30.0.0/24 | Consul discovery  |
+| Custom TCP Rule | TCP       | 4000       | 172.30.0.0/24 | Swarm HA managers |
+
+This example doesn't use the following rules. However, they are included here for your reference.
```

Comment:
Why are they included? Because they appear? Also, the "following rules is ambiguous" in this layouts...there is no list of reuls...there is a new line and another para following.